### ISSUE TITLE
Cant add any Spools, form to enter values is blocked by something

### ISSUE THREAD

**Issue by fenpaws** [OllisGit/OctoPrint-SpoolManager#162]
Heya, yeah sorry for that wired title but I have a strange issue.

## Systeminfo (OctoPrint)
[octoprint-systeminfo-20210515215315.zip](https://github.com/OllisGit/OctoPrint-SpoolManager/files/6488196/octoprint-systeminfo-20210515215315.zip)

## General description

When I want to add any Spools I have I only get a screen like this, it seems that something is blocking the form.
![grafik](https://user-images.githubusercontent.com/29946173/118377779-b9808f00-b5cf-11eb-986e-c0f49d45a2f1.png)

## What I tried/found out

I disabled every addon that would change the UI of OcoPrint, in my case OctoPrint-WideScreen and Themeify, nothing worked. 
After that tried a bit more and I found out after I delete this div it can see the form, but without any buttons to add it
<img width="1792" alt="grafik" src="https://user-images.githubusercontent.com/29946173/118377835-0f553700-b5d0-11eb-9465-fad7463933b1.png">

<img width="1792" alt="grafik" src="https://user-images.githubusercontent.com/29946173/118378032-306a5780-b5d1-11eb-83be-74dac40074d6.png">

**Comment by cjmalchow** [comment]
I am also having this same issue

**Comment by fenpaws** [comment]
@cjmalchow Yeah, I can confirm that. Somehow these two addons are incompatible.

**Comment by OllisGit** [comment]
Hi,
in the latest release V1.4.0 I did some UI-Improvements.

Please test and give me a feedback.

Thx,, in advance
Olli

**Comment by Xiondewah** [comment]
I've updated to the latest release but still have this issue.  I have disabled calibration companion and the issue resolves.
